tsconfig.json

```diff
@@ -18,8 +18,10 @@
   "compileOnSave": true,
   "include": [
     "src/entrypoint/*.ts",
-    "src/__tests__/*.ts",
+    "src/postman/*.ts",
+    "src/**/__tests__/*.ts",
+    "src/__mocks__/*.ts",
     "src/debug/*.ts",
-    "src/**/__tests__/.ts"
+    "src/*.d.ts",
```

Comment:
is there a reason to include mock and .d.ts in the include here? I would only the entry points might be needed

Reply:
Usually, `src/**/*` is included for typescript checking. `typescript-eslint` was not picking up on some files with the existing includes.

Reply:
here in my mind I had placed the entrypoints of the application. so that when we do the transpilation, only the necessary files are being "copied" over. 

maybe there should be no reason for files to exist if not imported eventually.